docs/source/search/features.rst

```diff
@@ -0,0 +1,74 @@
+Internal Search
+===============
+
+The Internal Search is a default SQL based engine implementation, created to support the most basic queries.
+While the performance and range of features might not be the same, depending on your instance, if your volume of
+records is small and retrieval needs are simple, this module should be enough.
+
+It supports the two types of records from a total of six targets where each query is made upon the respective title:
+
+- Events
+- Categories
+
+.. note::
+    The Internal Search does not yet support: contributions, sub-contributions, event notes and attachments.
```

Comment:
with indico/indico#4952 being ready, i think you could already update this (but mention that it only searches in titles/descriptions (and note content))